portfolio/src/main/webapp/script.js

```diff
@@ -53,10 +53,36 @@ function bubbleFall() {
  * @param {ArrayList<String>} comments
  */
 function getComments() {
-  fetch('/data').then(response => response.json()).then((comments) => {
+  const userNumComments = document.getElementById('number-of-comments').value;
+  let baseUrl = '/data?number-of-comments=';
+  let url = baseUrl.concat(userNumComments.toString());
+
+  fetch(url).then(response => response.json()).then((comments) => {
+    console.log(comments);
+
+    const commentSectionContainer = document.getElementById('comment-section-container');
+
+    for(const comment of comments) {
+      commentSectionContainer.appendChild(createCommentElement(comment));
+    }
+  });
+}
+
+/**
+ * Clear current comments.
+ * Get desired number of comments from the data servlet.
+ * @param {ArrayList<String>} comments
+ */
+function setNumComments() {
+  const userNumComments = document.getElementById('number-of-comments').value;
+  let baseUrl = '/data?number-of-comments=';
+  let url = baseUrl.concat(userNumComments.toString());
+
+  fetch(url).then(response => response.json()).then((comments) => {
     console.log(comments);
 
     const commentSectionContainer = document.getElementById('comment-section-container');
+    commentSectionContainer.innerHTML = '';
```

Comment:
Do we want to set `innerHTML` to empty in getComments as well? Or I guess, what does `setNumComments` do that we can't just use `getComments` for? Can we just have 1 function?

Reply:
You are completely right! Didn't realize those functions are 1 line different.